### PR TITLE
Yurii shevchuk

### PR DESCRIPTION
**Pull request by jurec72**
# Homework Stream
* Task 1
* Task 2
* Task 3

### CODE REVIEW

File: 02-GenericsAndLambda/src/main/java/homework/yurii_shevchuk/Dimenssion.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package homework.yurii_shevchuk;
+
+public class Dimenssion<T> {
+
```
Comment (mirza-05) [inline 02-GenericsAndLambda/src/main/java/homework/yurii_shevchuk/Dimenssion.java:4]: Remove extra Whitespace

File: 02-GenericsAndLambda/src/main/java/homework/yurii_shevchuk/Dimenssion.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+     T length;
+     T width;
+     T height;
+
+    public Dimenssion(T length, T width, T height) {
+        this.length = length;
+        this.width = width;
+        this.height = height;
+    }
+
```
Comment (mirza-05) [inline 02-GenericsAndLambda/src/main/java/homework/yurii_shevchuk/Dimenssion.java:14]: Remove extra Whitespace

File: 02-GenericsAndLambda/src/main/java/homework/yurii_shevchuk/Pair.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
+package homework.yurii_shevchuk;
+
+public class Pair<T1, T2> {
+
```
Comment (mirza-05) [inline 02-GenericsAndLambda/src/main/java/homework/yurii_shevchuk/Pair.java:4]: Remove extra Whitespace

File: 02-GenericsAndLambda/src/main/java/homework/yurii_shevchuk/PairTest.java [hunk 4]
```diff
@@ -0,0 +1,9 @@
+package homework.yurii_shevchuk;
+
+public class PairTest {
+
```
Comment (mirza-05) [inline 02-GenericsAndLambda/src/main/java/homework/yurii_shevchuk/PairTest.java:4]: Remove extra Whitespace

File: 02-GenericsAndLambda/src/main/java/homework/yurii_shevchuk/TaskGenerics.java [hunk 5]
```diff
@@ -0,0 +1,11 @@
+package homework.yurii_shevchuk;
+
+public class TaskGenerics {
+
+    public static void main(String[] args) {
+
```
Comment (mirza-05) [inline 02-GenericsAndLambda/src/main/java/homework/yurii_shevchuk/TaskGenerics.java:6]: Remove extra Whitespace

File: 03-GenericsLambdaStream/src/main/java/homework/yurii_shevchuk/StreamTasks.java [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+        Stream.of("2", 3, "Jack").filter(x -> x.equals("2")).forEach(x -> System.out.println(x));
+        Stream.of("2", 3, "Jack").limit(1).forEach(System.out::println);
+
+
+//         TASK 3
+        // todo print "the quick brown fox and blue hare."
+        //  from following string
+
+        String s = "the quick brown fox and the quick blue hare.";
+        Arrays.asList(s.split("\\s")).stream().distinct().forEach(x -> System.out.print(x+" "));
```
Comment (kykaparov) [inline 03-GenericsLambdaStream/src/main/java/homework/yurii_shevchuk/StreamTasks.java:36]: the text was "The quick brown fox and the quick blue hare.", "The" with uppercase T :)

**Pull request merged**